config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -1,11 +1,11 @@
 periodics:
 - annotations:
     testgrid-alert-stale-results-hours: "24"
-    testgrid-dashboards: sig-release-1.16-informing, conformance-all, conformance-gce
+    testgrid-dashboards: sig-release-1.16-blocking, conformance-all, conformance-gce
     testgrid-num-columns-recent: "3"
     testgrid-num-failures-to-alert: "1"
     testgrid-tab-name: GCE, 1.16 (dev)
-  interval: 6h
+  interval: 3h
```

Comment:
fwiw, these were historically arbitrary, we can certainly handle running them this often without issue.